pkg/apis/extensions/install/install.go

```diff
@@ -43,11 +43,9 @@ var availableVersions = []unversioned.GroupVersion{v1beta1.SchemeGroupVersion}
 
 func init() {
```

Comment:
Comment on availableVersions (line 42) and not this line:
The ordering comment there is invalid now right?